### PR TITLE
Pagination in Drilldown tables

### PR DESCRIPTION
**Pull request by CPAlejandro**
Hi guys,

In this PR we switch from basic tables to memory tables to be able to show the paginated tables.

image

To test it we will only have to get into the GitHub module and go to the Panel tab

### CODE REVIEW

File: public/components/overview/github-panel/config/main.tsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ export const MainViewConfig = {
                 tableTitle='Actors'
                 aggTerm='data.github.actor'
                 aggLabel='Actor'
-                maxRows={5} 
+                maxRows={100}
```
Comment (Desvelao) [inline public/components/overview/github-panel/config/main.tsx:31]: **thought**: I am not sure why this change.

**Review by sortizowlh — APPROVED** [review]
CR: 🟢 
Test: 🟢

**Comment by CPAlejandro** [comment]
**Final result:**

image

File: public/components/overview/github-panel/config/main.tsx (hunk 1 above)
Comment (CPAlejandro) [reply public/components/overview/github-panel/config/main.tsx]: I have modified it by a variable, to extract the same data from the 4 fields. Thanks for the warning.

File: public/components/common/modules/panel/components/agg-table.tsx [hunk 2]
```diff
@@ -56,18 +56,29 @@ export const AggTable = ({
       },
     };
   };
-
+  const sorting = {
```
Comment (mpRegalado) [inline public/components/common/modules/panel/components/agg-table.tsx:59]: Adding sorting to an EuiInMemoryTable without knowing that all the possible data is already in memory can lead to unaccurate results.
You could use `onTableChange` as [documented by Eui](https://elastic.github.io/eui/#/tabular-content/in-memory-tables) to change the value of `preAppliedAggs.buckets.terms.order` for either `_count` or `_key` to be `asc` or `desc` as described in the [Elastic doc](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-terms-aggregation.html#search-aggregations-bucket-terms-aggregation-order). It would just require mapping the `field` value to one of those.

**Review by mpRegalado — CHANGES_REQUESTED** [review]

**Comment by sortizowlh** [comment]
Tested all new changes ✔️ 
Now GitHub panel tab has 10 rows per table.

**Comment by sortizowlh** [comment]
Second revision. LGTM!
Now, all tables have 10 rows for GitHub Panel tab
